child_compassion/mappings/compassion_child_mapping.py

```diff
@@ -32,6 +32,54 @@ class GenericChildMapping(OnrampMapping):
         'GlobalId': 'global_id',
         'LocalBeneficiaryId': 'local_id',
 
+        'AcademicPerformance': 'academic_performance',
+        'ActiveProgram': 'type',
+        'AgeInYearsAndMonths': None,
+        'BeneficiaryStatus': 'state',
+        'Birthdate': 'estimated_birthdate',
+        'CDSPType': 'cdsp_type',
+        'ChristianActivities': (
+            'christian_activity_ids', 'child.christian.activity'),
+        'ChronicIllness': ('chronic_illness_ids', 'child.chronic.illness'),
+        'CognitiveAgeGroup': 'cognitive_age_group',
+        'CommunityName': ('project_id.community_name''compassion.project'),
+        'CompassBeneficiaryID': 'compass_id',
+        'CorrespondenceLanguage': 'correspondence_language',
+        'CurrentUniversityYearOfStudy': None,
+        'EstimatedBirthDate': 'estimated_birthdate',
+        'ExpectedTransitionDateToSponsorship': None,
+        'FO': ('project_id.name', 'compassion.project'),
+        'FavoriteProjectActivities': ('activity_ids',
+                                      'child.project.activity'),
+        'FavoriteSubjectInSchool': ('subject_ids', 'child.school.subject'),
+        'FormalEducationLevel': 'education_level',
+        'HeightInCm': 'height',
+        'HouseholdDuties': ('duty_ids', 'child.household.duty'),
+        'HouseholdInformation': ('household_id', 'compassion.household'),
+        'HouseholdMembersList': ('household_id.member_ids',
+                                 'compassion.household.member'),
+        'ICP': ('project_id.name', 'compassion.project'),
+        'ICPID': ('project_id.icp_id', 'compassion.project'),
+        'LastReviewedDate': 'last_review_date',
+        'LocalBeneficiaryNumber': None,
+        'LocalGradeLevel': 'local_grade_level',
+        'MajorCourseOfStudy': 'major_course_study',
+        'MentalDevelopmentConditions': None,
+        'NameInNonLatinChar': 'non_latin_name',
+        'NotEnrolledInEducationReason': 'not_enrolled_reason',
+        'PhysicalDisabilities': ('physical_disability_ids',
+                                 'child.physical.disability'),
+        'PlannedCompletionDate': 'completion_date',
+        'PlannedCompletionDateChangeReason': None,
+        'PrimaryCaregiverName': None,
+        'ReligiousAffiliation': None,
+        'SponsoredStatus': ('sponsor_id.states', 'res.partner'),
+        'ThingsILikeAge1To2': ('hobby_ids', 'child.hobby'),
+        'ThingsILikeAge3Plus': None,
+        'TypeOfVocationalEducation': 'vocational_training_type',
+        'USEquivalentGradeLevel': 'local_grade_level',
+        'WeightInKg': 'weight',
```

Comment:
You ordered fields alphabetically which is great :+1: 
We should do the same in all our mappings